docs/docsite/rst/network.rst

```diff
@@ -0,0 +1,15 @@
+Ansible Network
```

Comment:
```
***************
Ansible Network
***************
```

Reply:
Current docs have inconsistent header styling (it has probably accumulated through the years). I'd prefer to discuss this, come up with a consistent notation, and submit one PR that makes all the docs use the same notation.

Reply:
Created https://github.com/ansible/ansible/issues/35520 to address this.

Reply:
Agreed. Ignore this comment and we can address this in the bulk fix up PR.

As long as Shippable RST check passes  (which it is) I'm happy.